src/orquestra/sdk/_base/cli/_dorq/_repos.py

```diff
@@ -20,7 +20,17 @@
 from orquestra.sdk._base._driver._client import DriverClient
 from orquestra.sdk._base._qe import _client
 from orquestra.sdk.schema.configs import ConfigName, RuntimeName
-from orquestra.sdk.schema.workflow_run import WorkflowRun, WorkflowRunId
+from orquestra.sdk.schema.ir import (
+    FunctionRef,
+    TaskDefId,
+    TaskInvocationId,
+    WorkflowDef,
+)
+from orquestra.sdk.schema.workflow_run import TaskRunId, WorkflowRun, WorkflowRunId
+
+
+def _find_first(f: t.Callable[[t.Any], bool], it: t.Iterable):
```

Comment:
Thought: This is trivial function used once. Is it really needed? The signature confused me for a bit tbh 😂

Reply:
It communicates the intent better than the `next(filter(...))`. I believe it's worth it for the same reason why `map()` and comprehensions are better than the generic `for` loop.

Reply:
Other languages come with a standard library that includes a plethora of utilities for doing common things with sequences, like `find_first`, `window`, `batch`, etc. Python is kinda backwards in this regard 😄